frontend/packages/client/src/utils.js

```diff
@@ -1,3 +1,5 @@
+import { stateToHTML } from "draft-js-export-html";
```

Comment:
replaces `draftjs-to-html` with `draft-js-export-html` since it provides better functions for block mapping and  creating html